manager/state/raft/raft.go

```diff
@@ -342,6 +398,9 @@ func (n *Node) JoinAndStart(ctx context.Context) (err error) {
 	}
 
 	// join to existing cluster
+	if n.opts.Addr == "" {
```

Comment:
It does not need to be called before `Run`. See the new test `TestServiceCreateLateBind`, where it is called afterwards.